### ISSUE TITLE
make sure to comment the main parts of your code

### ISSUE THREAD

**Issue by ad3429** [ShawnManek/prj-rev-bwfs-dasmoto#2]
Even though your code is very easy to follow, it is a good idea to make sure to comment the main parts of your code. This becomes increasingly important as your projects become more complex. To comment a part of your code you can do the following:

![comment](https://user-images.githubusercontent.com/25671399/28217515-4e2987d6-6883-11e7-8111-bd88c83d72f8.png)

**Comment by ShawnManek** [comment]
I tried this in CSS and the comment wasn't working. does the comment have to be on 1 single line for the CSS stylesheet? 

Basically are there any specific notes when using the comment ability for stylesheets vs html?

**Comment by ad3429** [comment]
Yup, you are right -- this is the comment format for just HTML. For CSS, comments look like this:

for a single line comment:
//This is a comment

or, for a multiline comment, you would do:

/*
    This is a comment
*/
